src/tests/test_output.py

```diff
@@ -0,0 +1,43 @@
+import os
+import argparse
+import subprocess
+import pytest
+
+
+def extract_files_with_extension(dir_name, extension):
```

Comment:
This is duplicated code. It could live in a `support.py` (or similar name) file and be imported?

Reply:
Same for any other duplication.